### ISSUE TITLE
syndicate posts despite "Could not find a tweet to reply to."

### ISSUE THREAD

**Issue by dissolve** [snarfed/bridgy#362]
I often reply to websites but would like to post the reply to twitter, even though it is out of context.
Readers to follow the link to find out what i am talking about.

If I reply to a web-url and specifically tag it to syndicate to twitter, I would like it to still syndicate to twitter.

**Comment by dissolve** [comment]
As an example

https://www.brid.gy/log?start_time=1423487412&key=aglzfmJyaWQtZ3lyaAsSDVB1Ymxpc2hlZFBhZ2UiQWh0dHBzOi8vYmVuLnRoYXRtdXN0YmUubWUvcG9zdC8yMDE1LzIvOS8xL3NvY2lhbF9hcGlfdXNlcl9zdG9yaWVzDAsSB1B1Ymxpc2gYgICAgICAgAoM

**Comment by snarfed** [comment]
interesting idea! definitely sounds reasonable to support.

the catch is, people also often try to use bridgy publish to POSSE a reply to a tweet but get the in-reply-to link wrong, or accidentally use brid.gy/publish/twitter when they meant to use brid.gy/publish/facebook, etc. this check helps them. i'm not sure yet how to reconcile the two use cases.

**Comment by dissolve** [comment]
I think its better to assume the user is doing things correctly than to assume incorrect.

There are probably some situations that can be filtered.  If a in-reply-to is a twitter URL or Facebook URL, then perform by current methods, otherwise ignore the in-reply-to and post as a standard note.

**Comment by snarfed** [comment]
i wish that was true! sadly, there are dozens of bridgy publish attempts every day that are wrong in various ways, and mismatched in-reply-to and bridgy link is one of the most common. (i can tell because invariably the person fixes it and makes another attempt with the right in-reply-to or bridgy link minutes later.)

**Comment by snarfed** [comment]
one possible workaround is to publish your post without the in-reply-to link, trigger bridgy publish (via either webmention or the form on your user page), then add the in-reply-to link after. annoying, but it'd work.

**Comment by dissolve** [comment]
thats what i have done in the past, the most annoying is when it is on an article which i edit via html rather than just text (on notes etc) and I don't have edit functionality for it yet, so "Editing" an article post is actually just changed the DB directly

**Comment by tantek** [comment]
My analysis of this use-case (POSSEing a reply to something that is not a tweet nor has a POSSE tweet copy) is here, and is roughly what I've implemented in Falcon's code to POSSE reply posts to Twitter: https://indiewebcamp.com/Twitter#original_lacks_POSSE_tweet

**Comment by snarfed** [comment]
@singpolyma also votes for this for reposts in #555.

**Comment by singpolyma** [comment]
For the accident use case. Could add force_any_context=1 optional param (similar to include_link) to enable this

**Comment by kylewm** [comment]
I'm -1 on this proposal, per https://github.com/snarfed/bridgy/issues/527#issuecomment-158535263

It adds a lot of decision making (based on personal preferences) that Bridgy just isn't at the right layer to make.

**Comment by snarfed** [comment]
agreed, sorry. #527, definitely. i already regret the two user facing options bridgy publish does have, whether to include a link and (especially) whether to ignore formatting.

**Comment by kylewm** [comment]
to be clear, I'm -1 on this whole issue, not Stephen's `force_any_context` proposal specifically ;)
